packages/babel-generator/src/index.ts

```diff
@@ -71,7 +71,7 @@ function normalizeOptions(
       minimal: process.env.BABEL_8_BREAKING ? true : false,
       ...opts.jsescOption,
     },
-    recordAndTupleSyntaxType: opts.recordAndTupleSyntaxType,
+    recordAndTupleSyntaxType: opts.recordAndTupleSyntaxType || "hash",
```

Comment:
```suggestion
    recordAndTupleSyntaxType: opts.recordAndTupleSyntaxType ?? "hash",
```
boolean option here will not be accepted.